tuist/tuist
# Fix XCConfig path for swift package dependencies

Resolves https://github.com/tuist/tuist/issues/4443
Request for comments document (if applies):

### Short description 📝

Swift package dependencies were not picking XCConfigurations because of wrong path generated during `fetch` phase.

### How to test the changes locally 🧐

1. Use `ios_app_with_multi_configs` fixture project which contains a local swift package and try to use existing configuration in `ConfigurationFiles` 
2. Run `tuist fetch`
3. Run `tuist generate`
4. Generated project should have `LocalSwiftPackage` project which should be using XCConfigurations from `ConfigurationFiles` path

### Checklist ✅
